DiscordBot.DataAccess/Models/EventResponse.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace DiscordBot.DataAccess.Models
+{
+    public class EventResponse
+    {
+        public string emoji { get; }
+        public string responseName { get; }
```

Comment:
Should be named starting with uppercase as they're properties 🙂

Reply:
Yes, sorry! Fixed in 1718faa 🙂